### PR TITLE
SONARGRADL-7 Add automatic support of Groovy projects

### PR DESCRIPTION
**Pull request by henryju**

### CODE REVIEW

File: src/main/java/org/sonarqube/gradle/SonarQubePlugin.java [hunk 1]
```diff
@@ -213,73 +216,120 @@ private void configureForJava(final Project project, final Map<String, Object> p
...
+          if (addForGroovy) {
+            properties.put("sonar.groovy.jacoco.reportPath", destinationFile);
           }
-        });
+        }
       }
     });
   }
 
+  private void configureTestReports(Test testTask, Map<String, Object> properties) {
```
Comment (SonarTech) [inline src/main/java/org/sonarqube/gradle/SonarQubePlugin.java]: MAJOR Make "configureTestReports" a "static" method. [rule](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS2325)

**Comment by SonarTech** [comment]
SonarQube analysis reported 1 issue:
* MAJOR 1 major

Watch the comments in this conversation to review them.

**Comment by SonarTech** [comment]
SonarQube analysis reported 1 issue:
* MAJOR 1 major

Watch the comments in this conversation to review them.

**Comment by dbmeneses** [comment]
LGTM without going into too much detail (don't know gradle API)

**Pull request merged**